src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -0,0 +1,625 @@
+########################################################################
+# File :   Originally ARCComputingElement.py
+# Author : Raja Nandakumar
+# Use the ARC REST interface and retire api libraries when complete
+########################################################################
+
+""" AREX Computing Element (ARC REST interface)
+    Using the REST interface now and fail if REST interface is not available.
+    A lot of the features are common with the API interface. In particular, the XRSL
+    language is used in both cases. So, we retain the xrslExtraString and xrslMPExtraString strings.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+import six
```

Comment:
`six` belongs in the non-standard library imports and each list should be sorted alphabetically, i.e.
```python
import os
import stat
import sys

import json
import ldap3
import requests
import six
```

Reply:
Done.